test/calibration/test_calibrations.py

```diff
@@ -1292,19 +1293,18 @@ def test_get_parameter_value(self):
 class TestSavingAndLoading(CrossResonanceTest):
     """Test that calibrations can be saved and loaded to and from files."""
 
-    def tearDown(self):
-        """Clean-up"""
-        super().tearDown()
-
-        # Clean-up
-        os.remove("parameter_values.csv")
-        os.remove("parameter_config.csv")
-        os.remove("schedules.csv")
+    def _remove_files(self, prefix: str):
+        """Delete the files."""
+        os.remove(prefix + "parameter_values.csv")
+        os.remove(prefix + "parameter_config.csv")
+        os.remove(prefix + "schedules.csv")
 
     def test_save_load_parameter_values(self):
         """Test that we can save and load parameter values."""
 
-        self.cals.save("csv", overwrite=True)
+        prefix = str(uuid.uuid4())
```

Comment:
You could do this under `self.setUp()` and assign the prefix to an instance variable.

Reply:
Done here: 246ea96